cardano-cli/test/cardano-cli-golden/files/golden/help.cli

```diff
@@ -7003,7 +7003,8 @@ Usage: cardano-cli conway governance drep retirement-certificate
   Create a DRep retirement certificate.
 
 Usage: cardano-cli conway governance drep update-certificate 
-                                                               ( --drep-verification-key STRING
+                                                               ( --drep-script-hash HASH
```

Comment:
@CarlosLopezDeLara adding the new option on the command line to the PR description is useful because this then automatically ends up in the changelog